content/v2.0/write-data/use-telegraf/manual-config.md

```diff
@@ -53,11 +54,10 @@ For example: <code>https:<nolink>//us-west-2-1.aws.cloud2.influxdata.com</code>
 Your InfluxDB v2.0 authorization token.
 For information about viewing tokens, see [View tokens](/v2.0/security/tokens/view-tokens/).
 
-{{% note %}}
-#### Avoid storing tokens in plain text
+{{% warn %}}
+###### Avoid storing tokens in plain text
 InfluxData does not recommend storing authentication tokens in plain text in the `telegraf.conf`.
```

Comment:
We recommend storing your tokens securely by setting....and including the environment variable in....